WcaOnRails/app/models/competition.rb

```diff
@@ -142,10 +142,44 @@ def info_for(user)
     info
   end
 
+  attr_accessor :cloned_from_id
+  def cloned_from
+    Competition.find_by(id: cloned_from_id)
```

Comment:
That was my first thought, but this raises an error when there's no `cloned_from_id` whereas `find_by` returns nil in this case.